lang_qc/db/helper/wells.py

```diff
@@ -77,6 +80,44 @@ def get_mlwh_well_by_product_id(
             )
         ).scalar_one_or_none()
 
+    def get_metrics_by_well_product_id(
```

Comment:
Other QC metrics in `lang_qc/models/pacbio/qc_data.py` have class methods to self-populate themselves. It might be reasonable to move this code to such class method